common/src/main/java/org/jboss/hal/testsuite/page/config/ServerConfigurationPage.java

```diff
@@ -60,4 +45,18 @@ public ConfigFragment getWindowFragment() {
         WebElement editPanel = browser.findElement(By.className("default-window-content"));
         return  Graphene.createPageFragment(ConfigFragment.class, editPanel);
     }
+
+    public void navigate() {
+        browser.navigate().refresh();
+        Graphene.goTo(HomePage.class);
```

Comment:
This looks strange navigating to HomePage, when you are actually in ServerConfigurationPage. Is it really correct?